### ISSUE TITLE
Starting and stopping the notifications slash commands are not working

### ISSUE THREAD

**Issue by AayushChaudhary0001** [mattermost-community/mattermost-plugin-google-drive#5]
As a user, the notifications for commenting and sharing any file type in the drive are not working.

**Steps to reproduce:-**
1. Run the "/drive notifications start".
2. Share the desired file from the drive to a user and add a comment on it.
3. Go back to MM and open the bot channel.

The user did not receive the notification for the file type shared and comment added.

**Comment by darkLord19** [comment]
@AayushChaudhary0001 can you share details about your MM setup and steps to reproduce the issue because it was working in my testing. Just want to confirm that it's not some configuration issue before diving into debugging.

**Comment by wiggin77** [comment]
@AayushChaudhary0001 can you provide more information per above?

**Comment by AayushChaudhary0001** [comment]
@darkLord19 I am using an enterprise edition of Mattermost (version:- 7.1.0). Also, for plugin, I have created the build from master branch. I have already configured the client ID and client secret, as I am able to connect the account.

Steps to reproduce:-

1. Run the "/drive notifications start".
2. Share the desired file from the drive to a user and add a comment on it through the account of the user to whom the file is shared.
3. Go back to MM and open the bot channel of Google Drive.

What I did is share a document which was created in my drive to a user and add comments through that user's account. The notifications were already enabled still I didn't get any notification on MM instance.

I am also adding a supporting screenshot below.

![image](https://github.com/user-attachments/assets/c4f7642c-3bdc-4507-9e96-b4c9224cfaff)

**Comment by darkLord19** [comment]
@AayushChaudhary0001 did you comment using your own account? Comments made using the same account which is connected to mattermost won't trigger notifications.

**Comment by darkLord19** [comment]
You will need two accounts for this.
1. create a file using account linked to your mm profile
2. comment of that file using some other account

**Comment by AayushChaudhary0001** [comment]
@darkLord19 No, the account which I used for commenting is another user's account.

**Comment by darkLord19** [comment]
@AayushChaudhary0001 please use this branch https://github.com/mattermost-community/mattermost-plugin-google-drive/pull/12 and see if everything is working correctly.

Please see attached videos i have done clean setup on my device and it's working fine
https://drive.google.com/drive/folders/1iUckXDK_YKB2T48Ry2Cxa229dKpq6irB?usp=sharing

**Comment by AayushChaudhary0001** [comment]
@darkLord19 Tested PR #12 works fine for the login is a user comments, but there is an issue in the UI with the Indigo theme of Mattermost, can you please look into it?

I am attaching a screenshot as well for the same

![image](https://github.com/user-attachments/assets/958f292f-9ddc-420c-839e-7a5e83a69189)

Also, in other plugins blue coloured button is used in these type of attachments, if you can try to make it consistent in any way

![image](https://github.com/user-attachments/assets/b73a8d14-03b5-49c8-81a8-eacd121393ea)

**Comment by darkLord19** [comment]
@AayushChaudhary0001 can you close this issue and create a new one for theming issue?
